packages/bff-types-generated/src/index.ts

```diff
@@ -1 +1,18 @@
 export * from '../generated/sdk.ts';
+
+export enum DialogStatus {
```

Comment:
Add TODO (with link to issue) to remove this when schema from dialogporten is updated would be "tidy".